.github/workflows/build-test-package.yml

```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "abd38d5a0040b9a8fbb0ad3127089dbb72a93342"
+            itk-git-tag: "7548a390d71dc25b80e792703ee3cfea17853a99"
```

Comment:
Why this commit and not the RC3 tag?

Reply:
Windows builds run into: https://github.com/InsightSoftwareConsortium/ITK/commit/7cebc26dfae73d90dbf4702aa1ccc544e9b70a37